tests/shared/test_streamable_http.py

```diff
@@ -776,7 +779,8 @@ async def http_client(basic_server, basic_server_url):
 @pytest.fixture
 async def initialized_client_session(basic_server, basic_server_url):
     """Create initialized StreamableHTTP client session."""
-    async with streamablehttp_client(f"{basic_server_url}/mcp") as (
+    token = "TODO add token here"
+    async with streamablehttp_client(url=f"{basic_server_url}/mcp/", headers={'Authorization': f'Bearer {token}'}) as (
```

Comment:
Note: when running against the example server in the MCP repo, the client has to connect to /mcp/